libs/gltfio/src/MaterialGenerator.cpp

```diff
@@ -171,6 +171,21 @@ static Material* createMaterial(Engine* engine, const MaterialKey& config, const
             .material(shader.c_str())
             .doubleSided(config.doubleSided);
 
+    switch (engine->getBackend()) {
```

Comment:
Would be nice if we could just pass this directly to the MaterialBuilder.

Reply:
Yeah I think they are separate enums because MaterialBuilder has the concepts of "ALL" which Backend doesn't have, and Engine has the concept of "DEFAULT" and "NOOP" which MaterialBuilder doesn't have.